src/couch_replicator_httpc_pool.erl

```diff
@@ -147,9 +132,8 @@ code_change(_OldVsn, #state{}=State, _Extra) ->
     {ok, State}.
 
 
-terminate(_Reason, State) ->
-    lists:foreach(fun couch_replicator_connection:relinquish/1, State#state.free),
-    lists:foreach(fun couch_replicator_connection:relinquish/1, State#state.busy).
+terminate(_Reason, _State) ->
+    ok.
```

Comment:
No, because the connection gen_server monitors the client pool process and relinquishes the connections when it sees the `'DOWN'` message.